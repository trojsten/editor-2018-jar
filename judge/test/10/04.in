INT:
velkostmapy
14
INT:
dlzka
0
INT:
vyska
0
INT:
okuliare
0
VECTOR_STR:
mapa
14
*6&''VVWHCJ,2>
##########5.IR
#4$CH[[N7#SWK'
#N*QH-(>1#GDKV
#4@3.HBUW#&>@%
#O9L-0S@S#7&%W
##########;E&1
&0BJKEV4H*+O7R
[.$<%6FX3RIR/G
,C9KA01,V;U*6G
GY=09B-?4L?Z6P
=QY.0KD*TY@50=
S784GM*U95S9VN
MU(P3>)B=5<Y;E
