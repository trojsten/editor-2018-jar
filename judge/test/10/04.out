{"dlzka": 10, "vyska": 6}